django/core/checks/security/base.py

```diff
@@ -69,9 +69,10 @@
 
 W009 = Warning(
     "Your SECRET_KEY has less than %(min_length)s characters or less than "
-    "%(min_unique_chars)s unique characters. Please generate a long and random "
-    "SECRET_KEY, otherwise many of Django's security-critical features will be "
-    "vulnerable to attack." % {
+    "%(min_unique_chars)s unique characters, or it's prefixed with "
+    "'django-insecure-' indicating that it was generated automatically by "
+    "Django. Please generate a long and random SECRET_KEY, otherwise many of "
+    "Django's security-critical features will be vulnerable to attack." % {
```

Comment:
```suggestion
    "%(min_unique_chars)s unique characters, or it's prefixed with %(prefix)s "
    "indicating that it was generated automatically by Django. Please "
    "generate a long and random SECRET_KEY, otherwise many of Django's "
    "security-critical features will be vulnerable to attack." % {
```